MotionMark/unit-tests/resources/js-test.js

```diff
@@ -0,0 +1,968 @@
+// Copyright 2014 Google Inc.  All rights reserved.
```

Comment:
Unit tests are great but let's not add a bunch of testing code in a small incremental update.

Reply:
Unit test is removed